cli/cmd/upgrade.go

```diff
@@ -207,8 +208,8 @@ func doUpgrade() error {
 	}
 
 	if err := helm.NewHelper().UpgradeChart(keptnUpgradeChart, keptnReleaseName, keptnNamespace, nil); err != nil {
-		logging.PrintLog("Could not complete Keptn upgrade: "+err.Error(), logging.InfoLevel)
```

Comment:
Note: removed the `logging.Printlog` since the error message was printed twice otherwise.